lib/logstash/outputs/csv.rb

```diff
@@ -44,7 +44,11 @@ def receive(event)
   private
   def get_value(name, event)
     val = event[name]
-    val.is_a?(Hash) ? LogStash::Json.dump(val) : val
+    val.is_a?(Hash) ? LogStash::Json.dump(val) : escape_csv(val)
   end
-end # class LogStash::Outputs::CSV
 
+  private
+  def escape_csv(val)
+    val.is_a?(String) && val.start_with?("=") ? "'#{val}" : val
```

Comment:
Is it right to put a single quote in front like that? Seems more like to quote it you want to wrap it in quotes. like `"\"#{val}\""`